inc/umock_c/umock_c_internal.h

```diff
@@ -581,10 +580,14 @@ typedef int(*TRACK_DESTROY_FUNC_TYPE)(PAIRED_HANDLES* paired_handles, const void
         return mock_call_modifier; \
     } \
 
+
+#define IMPLEMENT_IGNORE_ALL_CALLS_FUNCTION_DECL(return_type, name, ...) \
+    MU_C2(mock_call_modifier_,name) MU_C2(ignore_all_calls_func_,name)(void); \
+
```

Comment:
Interesting that this has no Codes?

Reply:
Errr... lines 587 and 588 immediately following state: 

```
The IgnoreAllCalls call modifier shall ...
If no matching call occurs...
```

I think it has the "codes"